projects/canopy/src/lib/carousel/carousel.component.html

```diff
@@ -13,7 +13,7 @@
     class="lg-carousel__wrapper"
     [style.width.%]="100 * carouselItemCount"
     [style.left.%]="-100 * selectedItemIndex"
-    [style.transition]="'left ease ' + slideDuration / 1000 + 's'"
```

Comment:
It seems the latest versions of Angular don't take into account default values for inline styles, for example the the default value for the CSS property `transition-timing-function` is `ease`. Using it here is redundant.

Reply:
This assumption can be checked by using a different timing function. Will provide a screenshot soon.